### ISSUE TITLE
Composer error

### ISSUE THREAD

**Issue by petrk94** [crscheid/php-article-extractor#34]
Hi,

I get the following error by installing it with composer:

composer require crscheid/php-article-extractor
Using version ^2.2 for crscheid/php-article-extractor
./composer.json has been created
Running composer update crscheid/php-article-extractor
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - crscheid/php-article-extractor[2.2, ..., 2.2.1] require scotteh/php-goose dev-master -> found scotteh/php-goose[dev-master, 1.0.x-dev (alias of dev-master)] but it does not match your minimum-stability.
    - Root composer.json requires crscheid/php-article-extractor ^2.2 -> satisfiable by crscheid/php-article-extractor[2.2, 2.2.1].

Installation failed, deleting ./composer.json.

**Comment by crscheid** [comment]
It may be that your composer.json file has a minimum-stability that is greater than what this project allows. Do you mind posting a snippet of your composer.json file that is using this project as a dependency?

**Comment by kadnan** [comment]
@crscheid 
I have no existing `composer.json` exist yet  I am getting the issue:

```
Using version ^2.5 for crscheid/php-article-extractor
./composer.json has been created
Running composer update crscheid/php-article-extractor
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - crscheid/php-article-extractor[2.5, ..., 2.5.2] require scotteh/php-goose dev-master -> found scotteh/php-goose[dev-master, 1.0.x-dev (alias of dev-master)] but it does not match your minimum-stability.
    - Root composer.json requires crscheid/php-article-extractor ^2.5 -> satisfiable by crscheid/php-article-extractor[2.5, 2.5.1, 2.5.2].

You can also try re-running composer require with an explicit version constraint, e.g. "composer require crscheid/php-article-extractor:*" to figure out if any version is installable, or "composer require crscheid/php-article-extractor:^2.1" if you know which you need.
```

I am using PHP 7.4

**Comment by crscheid** [comment]
I realized you don't have a `composer.json` file, but if you try it with that approach try adding `"minimum-stability": "dev"` to your composer.json. In the meantime, I'll look at the dependency to see if there is a version of the `scotteh/php-goose` library that I can depend on instead.


```sh
# php --version
PHP 7.4.30 (cli) (built: Jun 23 2022 08:42:52) ( NTS )
Copyright (c) The PHP Group
Zend Engine v3.4.0, Copyright (c) Zend Technologies

# composer --version
Composer version 2.3.7 2022-06-06 16:43:28

# cat composer.json
{
    "require": {
	"crscheid/php-article-extractor": "2.5.2"
    },
    "minimum-stability": "dev"
}

# composer install

```

**Comment by crscheid** [comment]
I think if I fix the dependency to `1.1.0` for that library, that will do it. I'll try to get to this later today.

**Comment by crscheid** [comment]
A new version was posted `2.5.3` which locks the composer dependency to a specific version number. This should fix the issue.
